USF-1670: Dropins Localization (#95)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/pdp-dictionary.mdx

---

> title: Product Details Dictionary

Reviewer: ```suggestion
title: Product details page dictionary
```

> description: Reference the default dictionary for the Product Details (PDP) dropin.

Reviewer: ```suggestion
description: Learn how to customize the default dictionary for the product details page (PDP) drop-in component.
```

---

import CodeImport from '@components/CodeImport.astro';
import CodeInclude from '@components/CodeInclude.astro';
import dictionary from '@dropins/storefront-pdp/i18n/en_US.json.d.ts?raw';

> The PDP dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Product Details (PDP) drop-in.

Reviewer: ```suggestion
The product details page (PDP) dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the PDP drop-in component.
```
